CONTRIBUTING.md

```diff
@@ -1,118 +1,182 @@
+### Work from GitHub
 
-If you ask me for a recommendation, I would suggest PyCharm (complete
-IDE, free and gratis, RAM-hungry) or vim (powerful editor, very
-lightweight, steep learning curve). Other people use Spyder, emacs,
-gedit, Notepad++, Sublime, Atom\...
+GitHub makes it very easy to make small contributions
+directly from your browser, without having to install any additional software.
+To get familiar with the process, you can do
+[this interactive GitHub training](https://lab.github.com/githubtraining/introduction-to-github).
+Once you have finished it, you can edit the poliastro source files
+straight from GitHub and open pull requests from your browser.
 
-You will also need to understand how git works. git is a decentralized
+### Work locally
+
+For more complex contributions, it will be more effective
+to set up a local development environment and work from your own computer.
+
+The most important thing is to understand how git works. git is a decentralized
 version control system that preserves the history of the software, helps
 tracking changes and allows for multiple versions of the code to exist
-at the same time. If you are new to git and version control, I recommend
-following [the Try Git tutorial](https://try.github.io/).
-
-If you already know how all this works and would like to contribute new
-features then that\'s awesome! Before rushing out though please make
-sure it is within the scope of the library so you don\'t waste your time
--[the mailing list](https://groups.io/g/poliastro-dev) or [the
-chat](http://chat.poliastro.space/) are good places to ask.
-
-If the feature you suggest happens to be useful and within scope, you
-will probably be advised to create a new
-[wiki](https://github.com/poliastro/poliastro/wiki) page with some
-information about what problem you are trying to solve, how do you plan
-to do it and a sketch or idea of how the API is going to look like. You
-can go there to read other good examples on how to do it. The purpose is
-to describe without too much code what you are trying to accomplish to
-justify the effort and to make it understandable to a broader audience.
-
-All new features should be thoroughly tested, and in the ideal case the
-coverage rate should increase or stay the same. Automatic services will
-ensure your code works on all the operative systems and package
-combinations poliastro support - specifically, note that poliastro is a
-Python 3 only library.
-
-## Development environment
-
-These are some succint steps to set up a development environment:
+at the same time. To learn more, you can have a look at
+[this Getting Started guide](https://docs.github.com/en/get-started/getting-started-with-git).
+
+Finally, you might want to have an Integrated Development Environment (IDE)
+that makes editing the source files easier.
+If you are hesitant on what IDE or editor to use,
+have a look at PyCharm, Visual Studio Code, or Spyder.
```

Comment:
Would adding the official web links for the IDEs described here, be a good choice?

Reply:
Good idea, I was a bit lazy 😂 will do!